charts/karmada/templates/kube-controller-manager.yaml

```diff
@@ -57,8 +57,8 @@ spec:
             - --client-ca-file=/etc/karmada/pki/server-ca.crt
             - --cluster-cidr=10.244.0.0/16
             - --cluster-name=karmada
-            - --cluster-signing-cert-file=/etc/karmada/pki/karmada.crt
-            - --cluster-signing-key-file=/etc/karmada/pki/karmada.key
+            - --cluster-signing-cert-file=/etc/karmada/pki/server-ca.crt
+            - --cluster-signing-key-file=/etc/karmada/pki/server-ca.key
```

Comment:
Nice work, it refer https://github.com/karmada-io/karmada/pull/2097#discussion_r914557504

Reply:
We found this bug in testing